lib/oboe_metal.rb

```diff
@@ -98,7 +73,7 @@ def clear_all_traces
       # Retrieves all traces written to the trace file
       #
       def get_all_traces
-        io = File.open(TRACE_FILE, 'r')
+        io = File.open(AppOpticsAPM::OboeInitOptions.instance.host, 'r')
         contents = io.readlines(nil)
```

Comment:
don't need to close files in ruby? this https://stackoverflow.com/questions/4795447/rubys-file-open-and-the-need-for-f-close kind of indicates that one should but you know what's going on better than i do.

Reply:
I didn't write this code, I just replaced the file name ;)